bbp/nse/blue_repair_sdk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchgitExternal, stdenv, boost, cmake, pkgconfig, gsl, bbpsdk, hdf5-cpp, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "BlueRepairSDK-${version}";
```

Comment:
Same for the name here. 

If you could switch it to something without camelcase to keep it consistent with other packages.